src/assets/scss/iadc/_media.scss

```diff
@@ -1,9 +1,7 @@
 // Video iframe ===================================
 video {
-  padding: 10px;
   width: 70%;
   height: auto;
-  box-sizing: border-box;
```

Comment:
Why do you remove the box-sizing attribute from this "iadc" theme and not from the default one?

Reply:
My bad. 
Even though the border-box have no visual effect without the padding, i'm gonna remove it so the two files match.